designer/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala

```diff
@@ -191,7 +193,9 @@ trait NusskanckerDefaultAppRouter extends NusskanckerAppRouter {
           processResolving = processResolving,
           processAuthorizer = processAuthorizer,
           processChangeListener = processChangeListener,
-          typeToConfig = typeToConfig
+          typeToConfig = typeToConfig,
+          categoryService = processCategoryService,
+          stateDefinitionService = statusDefinitionService
```

Comment:
@arkadius state? status?

Reply:
You ask me if it should be stateDefinitionService or statusDefinitionService? I'm not sure, pick your best choice.